### ISSUE TITLE
Manifest contains unhelpful items

### ISSUE THREAD

**Issue by GrahamBorland** [0ximDigital/RxAnimations#9]
I just tried to add RxAnimations 0.9.0 to my project, and I immediately started getting build errors.

```
Error:
	Attribute application@supportsRtl value=(false) from AndroidManifest.xml:22:9-36
	is also present at [oxim.digital:rxanim:0.9.0] AndroidManifest.xml:14:9-35 value=(true).
	Suggestion: add 'tools:replace="android:supportsRtl"' to <application> element at AndroidManifest.xml:19:5-96:19 to override.
```
This is because the library's AndroidManifest.xml includes values for `allowBackup` and `supportsRtl` which don't belong in a library project.

```xml
android:allowBackup="true"
android:label="@string/app_name"
android:supportsRtl="true"
```

**Comment by GrahamBorland** [comment]
This applies to Rx2Animations as well.

**Comment by 0ximDigital** [comment]
Hi gborland.

Thank you for trying out RxAnimations and raising the issue 🚀 

This will be fixed asap for the RxAnimations and Rx2Animations as well.

**Comment by 0ximDigital** [comment]
Fixed with 0.9.1 version for both RxAnimations and Rx2Animations.
